ruby/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM socrata/base
+MAINTAINER Socrata <sysadmin@socrata.com>
+
+RUN DEBIAN_FRONTEND=noninteractive apt-get -y update && \
+  DEBIAN_FRONTEND=noninteractive apt-get --force-yes -fuy install software-properties-common && \
+  DEBIAN_FRONTEND=noninteractive add-apt-repository -y ppa:brightbox/ruby-ng && \
+  DEBIAN_FRONTEND=noninteractive apt-get -y update && \
+  DEBIAN_FRONTEND=noninteractive apt-get install -y ruby2.1 && \
+  DEBIAN_FRONTEND=noninteractive apt-get purge -y --auto-remove software-properties-common && \
+  rm -rf /var/lib/apt/lists/*
+
+# skip installing gem documentation
+RUN echo 'gem: --no-rdoc --no-ri' >> "$HOME/.gemrc"
```

Comment:
```
$ docker run -it socrata/base bash
root@77a226163388:/# echo $HOME
/root
root@77a226163388:/#
```

Now, if a project wants to run as the 'socrata' user that's been added to the base image, it'll need to handle setting its own gem mumbo-jumbo.